android/app/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'au.com.robin.sms'
```

Comment:
`au.com.robin.sms` is hardcoded every where. Can you figure out how to make it like a variable instead. I want to be able to change this all in one place in the future.

Reply:
yes, I will be able to do that